gameplay/src/PlatformLinux.cpp

```diff
@@ -1333,6 +1334,13 @@ int Platform::enterMessagePump()
             _game->frame();
         }
 
+        if (!visible && __vsync) {
+        	// glXSwapBuffers may not block on vsync properly if the window is obscured/offdesktop --
+        	// if vsync enabled, do an artificial slowdown - 15ms is largely arbitrary
+        	// if the we have disabled vsync, we don't care, run full speed anyway.
+        	usleep(15000.0);
```

Comment:
I haven't tried it myself yet, but why not wait for a MapNotify event once we get a UnmapNotify event? Like it's done here: https://tronche.com/gui/x/xlib-tutorial/2nd-program-anatomy.html